service/popm/popm.go

```diff
@@ -167,6 +169,24 @@ func NewMiner(cfg *Config) (*Miner, error) {
 	return m, nil
 }
 
+// Fee returns the current fee in sats/vB used by the PoP Miner when
+// creating PoP transactions.
+func (m *Miner) Fee() uint {
+	return uint(m.txFee.Load())
+}
+
+// SetFee sets the fee in sats/vB used by the PoP Miner when creating
+// PoP transactions.
+func (m *Miner) SetFee(fee uint) {
+	switch {
+	case fee < 0:
+		fee = 1
+	case fee > 1<<32-1:
+		fee = 1<<32 - 1
```

Comment:
This is needed to safely convert `uint` to `uint32` on 64 bit platforms, even though the fee should never be anywhere near this value.